cmd/argocd/commands/app.go

```diff
@@ -730,6 +732,7 @@ func NewApplicationSetCommand(clientOpts *argocdclient.ClientOptions) *cobra.Com
 	var (
 		appOpts      cmdutil.AppOptions
 		appNamespace string
+		source_index int
```

Comment:
Just a nit, but this should be `sourceIndex` instead of `source_index` (also, throughout all other files, too)